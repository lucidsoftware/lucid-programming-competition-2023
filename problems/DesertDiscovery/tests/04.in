20
Sp8ayIDkExEXy9RNKbTJQa3jL3moPGSIQy
4
AteiCBCPajJWWuJPOyws4oTVbwyRt9KiWO
At9OpA3LKzSV3lf7yKhzYqgjTxhGNeVrkP
AtMEMYmC0wmLfGqWz0VNKvgh8zYKIj3CPt
AtToxAPj90HCcsK3moSgI3o0crnbsHobds
SpL1fo1ECbKNATkUqwM92wURsfCXdABTWk
2
AtrJv6ovnk9d3um1wL9xxvpTDSN1dlAvye
AtCWPR0rc641qxnnEz5Dgya4qLjUwuFwq4
SpRMi4AivEaSH6KxPYdnOag8qu9QbpX8xU
6
AtXhfiaPQWxh3HFdqQ9pUVVgFaCXdbxkEN
AtMEMYmC0wmLfGqWz0VNKvgh8zYKIj3CPt
AtJGDGtedXC1Xu0kzWIlhcQdS9Wrllb6Uj
AtF5J7AFZ6szWGSfuDG1lW0lZwFMD7jEpk
AtwnvzzUa9QqjMYngJq64xx96Eqw32m38C
At48gEnBJA24xUrjcseQuxGRfY6kJUIoxA
Sp0FgAQDoiZKHcXw3JxeetxjtSgB6cL05u
4
AtwnvzzUa9QqjMYngJq64xx96Eqw32m38C
AtIbeXScdKXw9KtZyyUY10lDftKDISj3Xm
AteiCBCPajJWWuJPOyws4oTVbwyRt9KiWO
At9OpA3LKzSV3lf7yKhzYqgjTxhGNeVrkP
SpGIgBM88cv7A8MbZpVOqXxMGOptcFT0iX
6
AtXhfiaPQWxh3HFdqQ9pUVVgFaCXdbxkEN
AtnEMpDww6f3AcTPQICZKMd6NxJp1XYa9J
AtJGDGtedXC1Xu0kzWIlhcQdS9Wrllb6Uj
AtCWPR0rc641qxnnEz5Dgya4qLjUwuFwq4
AtMEMYmC0wmLfGqWz0VNKvgh8zYKIj3CPt
At8ZqRwIpeVkvhbNUnenacJuYOj4S558ah
SpR3xfq019K5k7uVvZTmvNH4QCuq5wNaFo
4
AtIbeXScdKXw9KtZyyUY10lDftKDISj3Xm
AtrJv6ovnk9d3um1wL9xxvpTDSN1dlAvye
At48gEnBJA24xUrjcseQuxGRfY6kJUIoxA
AtDNQ0cDU3yi6kO8z9EluEQVznHmzPcCDx
Sp92yL3EXPOo4V0gtOgdqjwBphRLzfPNmC
5
At8ZqRwIpeVkvhbNUnenacJuYOj4S558ah
AteiCBCPajJWWuJPOyws4oTVbwyRt9KiWO
AtDNQ0cDU3yi6kO8z9EluEQVznHmzPcCDx
AtCWPR0rc641qxnnEz5Dgya4qLjUwuFwq4
AtIbeXScdKXw9KtZyyUY10lDftKDISj3Xm
SpKTHE2XNKBr7lXjyZiXHtEWe1eahrpU0I
6
At9OpA3LKzSV3lf7yKhzYqgjTxhGNeVrkP
AtJGDGtedXC1Xu0kzWIlhcQdS9Wrllb6Uj
AtDNQ0cDU3yi6kO8z9EluEQVznHmzPcCDx
AtCWPR0rc641qxnnEz5Dgya4qLjUwuFwq4
At48gEnBJA24xUrjcseQuxGRfY6kJUIoxA
AtnEMpDww6f3AcTPQICZKMd6NxJp1XYa9J
SpqylK81dMxDhGg8yo79ZUF5wApK8UoZc3
6
At9OpA3LKzSV3lf7yKhzYqgjTxhGNeVrkP
AtJGDGtedXC1Xu0kzWIlhcQdS9Wrllb6Uj
AteiCBCPajJWWuJPOyws4oTVbwyRt9KiWO
AtwnvzzUa9QqjMYngJq64xx96Eqw32m38C
At8ZqRwIpeVkvhbNUnenacJuYOj4S558ah
AtnEMpDww6f3AcTPQICZKMd6NxJp1XYa9J
SpFdy27FPUCBpeE3qkDQX3geqPvgqk1uDl
5
AtJGDGtedXC1Xu0kzWIlhcQdS9Wrllb6Uj
AtIbeXScdKXw9KtZyyUY10lDftKDISj3Xm
AtwnvzzUa9QqjMYngJq64xx96Eqw32m38C
AtDNQ0cDU3yi6kO8z9EluEQVznHmzPcCDx
At48gEnBJA24xUrjcseQuxGRfY6kJUIoxA
SpC5xGdIMVvcAhddTgleAyZxrrCLOIMBl2
4
AtDNQ0cDU3yi6kO8z9EluEQVznHmzPcCDx
AtrJv6ovnk9d3um1wL9xxvpTDSN1dlAvye
AtCWPR0rc641qxnnEz5Dgya4qLjUwuFwq4
AtwnvzzUa9QqjMYngJq64xx96Eqw32m38C
SpmcNhDHYJBO80c6qgotIKiNFqkVQ5y4xt
6
AteiCBCPajJWWuJPOyws4oTVbwyRt9KiWO
AtCWPR0rc641qxnnEz5Dgya4qLjUwuFwq4
At9OpA3LKzSV3lf7yKhzYqgjTxhGNeVrkP
AtToxAPj90HCcsK3moSgI3o0crnbsHobds
AtJGDGtedXC1Xu0kzWIlhcQdS9Wrllb6Uj
At48gEnBJA24xUrjcseQuxGRfY6kJUIoxA
SpvEEYcOatXZHeh02QGet4H29ZFpHSVa3C
4
AtF5J7AFZ6szWGSfuDG1lW0lZwFMD7jEpk
AtToxAPj90HCcsK3moSgI3o0crnbsHobds
AteiCBCPajJWWuJPOyws4oTVbwyRt9KiWO
AtJGDGtedXC1Xu0kzWIlhcQdS9Wrllb6Uj
SpG5gc27zcvZ3eaejgsKVvxk2x6mSEHH0V
2
AtSEv4OzwJFT3tVsw8FECijIC62vedLcM3
AtXhfiaPQWxh3HFdqQ9pUVVgFaCXdbxkEN
SpPwVKk6eVnSre7wfrENrjq4LuwjEOJekz
2
At48gEnBJA24xUrjcseQuxGRfY6kJUIoxA
AtMEMYmC0wmLfGqWz0VNKvgh8zYKIj3CPt
SpkIr1OO7Z17abJf486NIF6C4R1Pl3BhrV
3
AtToxAPj90HCcsK3moSgI3o0crnbsHobds
AtJGDGtedXC1Xu0kzWIlhcQdS9Wrllb6Uj
At9OpA3LKzSV3lf7yKhzYqgjTxhGNeVrkP
Sp4vvRXprDKYd0e1lwTUcLMKTFwj4Xj8hI
6
AtCWPR0rc641qxnnEz5Dgya4qLjUwuFwq4
AtF5J7AFZ6szWGSfuDG1lW0lZwFMD7jEpk
AtnEMpDww6f3AcTPQICZKMd6NxJp1XYa9J
AtXhfiaPQWxh3HFdqQ9pUVVgFaCXdbxkEN
AtSEv4OzwJFT3tVsw8FECijIC62vedLcM3
At48gEnBJA24xUrjcseQuxGRfY6kJUIoxA
SpXizm5BNdh1BntTKtwXMzbHdroZXdGK6f
5
AtJGDGtedXC1Xu0kzWIlhcQdS9Wrllb6Uj
At8ZqRwIpeVkvhbNUnenacJuYOj4S558ah
At48gEnBJA24xUrjcseQuxGRfY6kJUIoxA
AtIbeXScdKXw9KtZyyUY10lDftKDISj3Xm
AtF5J7AFZ6szWGSfuDG1lW0lZwFMD7jEpk
SpU1fBILAjNtwljgLEvDcD1E62PxtCXEkL
6
AtXhfiaPQWxh3HFdqQ9pUVVgFaCXdbxkEN
At48gEnBJA24xUrjcseQuxGRfY6kJUIoxA
At9OpA3LKzSV3lf7yKhzYqgjTxhGNeVrkP
AtToxAPj90HCcsK3moSgI3o0crnbsHobds
AtJGDGtedXC1Xu0kzWIlhcQdS9Wrllb6Uj
AtwnvzzUa9QqjMYngJq64xx96Eqw32m38C
SpkLWNR26qVKwvg2hblvm2HB5IwBrlBUsy
5
AtToxAPj90HCcsK3moSgI3o0crnbsHobds
AtSEv4OzwJFT3tVsw8FECijIC62vedLcM3
AtMEMYmC0wmLfGqWz0VNKvgh8zYKIj3CPt
AtrJv6ovnk9d3um1wL9xxvpTDSN1dlAvye
AtIbeXScdKXw9KtZyyUY10lDftKDISj3Xm
30
3
AtnEMpDww6f3AcTPQICZKMd6NxJp1XYa9J
AtXhfiaPQWxh3HFdqQ9pUVVgFaCXdbxkEN
AtToxAPj90HCcsK3moSgI3o0crnbsHobds
2
AtDNQ0cDU3yi6kO8z9EluEQVznHmzPcCDx
AtMEMYmC0wmLfGqWz0VNKvgh8zYKIj3CPt
4
AtwnvzzUa9QqjMYngJq64xx96Eqw32m38C
AtDNQ0cDU3yi6kO8z9EluEQVznHmzPcCDx
AtXhfiaPQWxh3HFdqQ9pUVVgFaCXdbxkEN
AtnEMpDww6f3AcTPQICZKMd6NxJp1XYa9J
2
AtJGDGtedXC1Xu0kzWIlhcQdS9Wrllb6Uj
AtDNQ0cDU3yi6kO8z9EluEQVznHmzPcCDx
5
AtCWPR0rc641qxnnEz5Dgya4qLjUwuFwq4
AtToxAPj90HCcsK3moSgI3o0crnbsHobds
AtSEv4OzwJFT3tVsw8FECijIC62vedLcM3
At9OpA3LKzSV3lf7yKhzYqgjTxhGNeVrkP
AtJGDGtedXC1Xu0kzWIlhcQdS9Wrllb6Uj
6
AtXhfiaPQWxh3HFdqQ9pUVVgFaCXdbxkEN
AtMEMYmC0wmLfGqWz0VNKvgh8zYKIj3CPt
AtSEv4OzwJFT3tVsw8FECijIC62vedLcM3
At48gEnBJA24xUrjcseQuxGRfY6kJUIoxA
AtwnvzzUa9QqjMYngJq64xx96Eqw32m38C
AtF5J7AFZ6szWGSfuDG1lW0lZwFMD7jEpk
6
AteiCBCPajJWWuJPOyws4oTVbwyRt9KiWO
AtIbeXScdKXw9KtZyyUY10lDftKDISj3Xm
AtXhfiaPQWxh3HFdqQ9pUVVgFaCXdbxkEN
AtDNQ0cDU3yi6kO8z9EluEQVznHmzPcCDx
AtF5J7AFZ6szWGSfuDG1lW0lZwFMD7jEpk
At48gEnBJA24xUrjcseQuxGRfY6kJUIoxA
2
AtXhfiaPQWxh3HFdqQ9pUVVgFaCXdbxkEN
AtwnvzzUa9QqjMYngJq64xx96Eqw32m38C
4
AtwnvzzUa9QqjMYngJq64xx96Eqw32m38C
At48gEnBJA24xUrjcseQuxGRfY6kJUIoxA
AtToxAPj90HCcsK3moSgI3o0crnbsHobds
AtIbeXScdKXw9KtZyyUY10lDftKDISj3Xm
6
AtCWPR0rc641qxnnEz5Dgya4qLjUwuFwq4
AtwnvzzUa9QqjMYngJq64xx96Eqw32m38C
AtrJv6ovnk9d3um1wL9xxvpTDSN1dlAvye
AtDNQ0cDU3yi6kO8z9EluEQVznHmzPcCDx
AtnEMpDww6f3AcTPQICZKMd6NxJp1XYa9J
AteiCBCPajJWWuJPOyws4oTVbwyRt9KiWO
3
AtMEMYmC0wmLfGqWz0VNKvgh8zYKIj3CPt
At8ZqRwIpeVkvhbNUnenacJuYOj4S558ah
At9OpA3LKzSV3lf7yKhzYqgjTxhGNeVrkP
4
AtIbeXScdKXw9KtZyyUY10lDftKDISj3Xm
AtF5J7AFZ6szWGSfuDG1lW0lZwFMD7jEpk
AtCWPR0rc641qxnnEz5Dgya4qLjUwuFwq4
AtnEMpDww6f3AcTPQICZKMd6NxJp1XYa9J
6
AtIbeXScdKXw9KtZyyUY10lDftKDISj3Xm
AtnEMpDww6f3AcTPQICZKMd6NxJp1XYa9J
At8ZqRwIpeVkvhbNUnenacJuYOj4S558ah
AtF5J7AFZ6szWGSfuDG1lW0lZwFMD7jEpk
AteiCBCPajJWWuJPOyws4oTVbwyRt9KiWO
AtJGDGtedXC1Xu0kzWIlhcQdS9Wrllb6Uj
5
At8ZqRwIpeVkvhbNUnenacJuYOj4S558ah
At48gEnBJA24xUrjcseQuxGRfY6kJUIoxA
AtnEMpDww6f3AcTPQICZKMd6NxJp1XYa9J
At9OpA3LKzSV3lf7yKhzYqgjTxhGNeVrkP
AtwnvzzUa9QqjMYngJq64xx96Eqw32m38C
4
AtIbeXScdKXw9KtZyyUY10lDftKDISj3Xm
AtnEMpDww6f3AcTPQICZKMd6NxJp1XYa9J
At8ZqRwIpeVkvhbNUnenacJuYOj4S558ah
AtwnvzzUa9QqjMYngJq64xx96Eqw32m38C
2
AtwnvzzUa9QqjMYngJq64xx96Eqw32m38C
At9OpA3LKzSV3lf7yKhzYqgjTxhGNeVrkP
6
AtXhfiaPQWxh3HFdqQ9pUVVgFaCXdbxkEN
AtSEv4OzwJFT3tVsw8FECijIC62vedLcM3
AtF5J7AFZ6szWGSfuDG1lW0lZwFMD7jEpk
At9OpA3LKzSV3lf7yKhzYqgjTxhGNeVrkP
AtnEMpDww6f3AcTPQICZKMd6NxJp1XYa9J
AtMEMYmC0wmLfGqWz0VNKvgh8zYKIj3CPt
5
AtnEMpDww6f3AcTPQICZKMd6NxJp1XYa9J
AtwnvzzUa9QqjMYngJq64xx96Eqw32m38C
At9OpA3LKzSV3lf7yKhzYqgjTxhGNeVrkP
AtF5J7AFZ6szWGSfuDG1lW0lZwFMD7jEpk
AtrJv6ovnk9d3um1wL9xxvpTDSN1dlAvye
6
AteiCBCPajJWWuJPOyws4oTVbwyRt9KiWO
AtF5J7AFZ6szWGSfuDG1lW0lZwFMD7jEpk
At48gEnBJA24xUrjcseQuxGRfY6kJUIoxA
AtMEMYmC0wmLfGqWz0VNKvgh8zYKIj3CPt
AtIbeXScdKXw9KtZyyUY10lDftKDISj3Xm
AtXhfiaPQWxh3HFdqQ9pUVVgFaCXdbxkEN
4
AtDNQ0cDU3yi6kO8z9EluEQVznHmzPcCDx
AtXhfiaPQWxh3HFdqQ9pUVVgFaCXdbxkEN
AtCWPR0rc641qxnnEz5Dgya4qLjUwuFwq4
AtSEv4OzwJFT3tVsw8FECijIC62vedLcM3
6
AtwnvzzUa9QqjMYngJq64xx96Eqw32m38C
At48gEnBJA24xUrjcseQuxGRfY6kJUIoxA
AtToxAPj90HCcsK3moSgI3o0crnbsHobds
AtXhfiaPQWxh3HFdqQ9pUVVgFaCXdbxkEN
AtSEv4OzwJFT3tVsw8FECijIC62vedLcM3
AtJGDGtedXC1Xu0kzWIlhcQdS9Wrllb6Uj
2
AtF5J7AFZ6szWGSfuDG1lW0lZwFMD7jEpk
AtIbeXScdKXw9KtZyyUY10lDftKDISj3Xm
6
At8ZqRwIpeVkvhbNUnenacJuYOj4S558ah
At9OpA3LKzSV3lf7yKhzYqgjTxhGNeVrkP
AtrJv6ovnk9d3um1wL9xxvpTDSN1dlAvye
AtMEMYmC0wmLfGqWz0VNKvgh8zYKIj3CPt
AtCWPR0rc641qxnnEz5Dgya4qLjUwuFwq4
AtJGDGtedXC1Xu0kzWIlhcQdS9Wrllb6Uj
2
AtDNQ0cDU3yi6kO8z9EluEQVznHmzPcCDx
At8ZqRwIpeVkvhbNUnenacJuYOj4S558ah
4
At9OpA3LKzSV3lf7yKhzYqgjTxhGNeVrkP
At8ZqRwIpeVkvhbNUnenacJuYOj4S558ah
AtIbeXScdKXw9KtZyyUY10lDftKDISj3Xm
AtwnvzzUa9QqjMYngJq64xx96Eqw32m38C
3
AtwnvzzUa9QqjMYngJq64xx96Eqw32m38C
AtToxAPj90HCcsK3moSgI3o0crnbsHobds
AtXhfiaPQWxh3HFdqQ9pUVVgFaCXdbxkEN
6
AtToxAPj90HCcsK3moSgI3o0crnbsHobds
AtDNQ0cDU3yi6kO8z9EluEQVznHmzPcCDx
AtJGDGtedXC1Xu0kzWIlhcQdS9Wrllb6Uj
AteiCBCPajJWWuJPOyws4oTVbwyRt9KiWO
AtIbeXScdKXw9KtZyyUY10lDftKDISj3Xm
At8ZqRwIpeVkvhbNUnenacJuYOj4S558ah
4
AtToxAPj90HCcsK3moSgI3o0crnbsHobds
AtMEMYmC0wmLfGqWz0VNKvgh8zYKIj3CPt
AtSEv4OzwJFT3tVsw8FECijIC62vedLcM3
AtIbeXScdKXw9KtZyyUY10lDftKDISj3Xm
4
AtrJv6ovnk9d3um1wL9xxvpTDSN1dlAvye
AtF5J7AFZ6szWGSfuDG1lW0lZwFMD7jEpk
AtwnvzzUa9QqjMYngJq64xx96Eqw32m38C
AtJGDGtedXC1Xu0kzWIlhcQdS9Wrllb6Uj
2
AtSEv4OzwJFT3tVsw8FECijIC62vedLcM3
AtwnvzzUa9QqjMYngJq64xx96Eqw32m38C
