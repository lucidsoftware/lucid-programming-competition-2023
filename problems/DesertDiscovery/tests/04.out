Possible New Discovery
Possible New Discovery
Possible New Discovery
Possible New Discovery
SpmcNhDHYJBO80c6qgotIKiNFqkVQ5y4xt
SpRMi4AivEaSH6KxPYdnOag8qu9QbpX8xU
Possible New Discovery
Possible New Discovery
Possible New Discovery
SpC5xGdIMVvcAhddTgleAyZxrrCLOIMBl2
Possible New Discovery
Possible New Discovery
Possible New Discovery
SpqylK81dMxDhGg8yo79ZUF5wApK8UoZc3
Possible New Discovery
Sp0FgAQDoiZKHcXw3JxeetxjtSgB6cL05u
Possible New Discovery
Possible New Discovery
Possible New Discovery
Possible New Discovery
SpU1fBILAjNtwljgLEvDcD1E62PxtCXEkL
Possible New Discovery
Possible New Discovery
Possible New Discovery
Sp0FgAQDoiZKHcXw3JxeetxjtSgB6cL05u
SpU1fBILAjNtwljgLEvDcD1E62PxtCXEkL
Possible New Discovery
SpkLWNR26qVKwvg2hblvm2HB5IwBrlBUsy
Possible New Discovery
Possible New Discovery
